src/Matterhorn/Types.hs

```diff
@@ -251,6 +251,7 @@ module Matterhorn.Types
   , userPrefTeamOrder
   , dmChannelShowPreference
   , groupChannelShowPreference
+  , favoriteChannelPreference
```

Comment:
Thank you for renaming this! In the future, please make changes like this in separate commits. The rename of this function is not related to the addition of `toggleChannelFavoriteStatus`. Having these changes in the same commit usually means they are related in some way, but as far as I can tell they could be committed separately.

Reply:
Noted.